cpg_infra/config/config.py

```diff
@@ -123,6 +123,10 @@ class GCP(DeserializableDataclass):
         gcp: GCP
         slack_channel: str | None = None
         etl_accessors: list[str] = dataclasses.field(default_factory=list)
+        etl_environment: str | None = None
+        etl_default_config: dict[str, str] | None = None
+        etl_private_repo_url: str | None = None
+        etl_private_repo_packages: str | None = None
```

Comment:
Is this a comma separated list of packages to install, what would you expect here?

Reply:
Change the packages to a list, more clearer this way.